src/tokenizer.cpp

```diff
@@ -327,61 +327,48 @@ static void end_float_token(Tokenize *t) {
     }
 
     // A SoftFloat-3e float128 is represented internally as a standard
-    // quad-precision float with 15bit exponent and 113bit fractional.
+    // quad-precision float with 15bit exponent and 112bit fractional.
     union { uint64_t repr[2]; float128_t actual; } f_bits;
 
     if (bigint_cmp_zero(&t->significand) == CmpEQ) {
         f_bits.repr[0] = 0;
         f_bits.repr[1] = 0;
     } else {
-        // normalize the significand
-        if (t->radix == 10) {
```

Comment:
We are handling this at the function entry point so this is irrelevant. May restructure if we stop using `strtod`, however.